tests/gnrc_tcp_client/Makefile

```diff
@@ -1,10 +1,13 @@
 include ../Makefile.tests_common
 
 # If no BOARD is found in the environment, use this default:
-BOARD ?= native
+BOARD = native
 
 ifeq (native,$(BOARD))
   PORT ?= tap1
+else
+  PORT = /dev/ttyACM1
+  SERIAL = <ADD_SERIAL>
```

Comment:
I'd suggest to remove this as well (here and below). It's always the same situation when you connect two boards